test/TestMatchError.py

```diff
@@ -7,6 +7,7 @@
 
 
 def test_matcherror_compare():
+"""Check that MatchError instances with similar attrs are equivalent."""
```

Comment:
Looks like I missed the indentation here
```suggestion
    """Check that MatchError instances with similar attrs are equivalent."""
```